spec/requests/requests_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe 'managing topics', :type => :request do
+
+  let!(:user) {FactoryBot.create :user}
+
+  it 'redirects unauthenticated users' do
+    post "/topics", :params => { :topic => {:name => "here is a new topic"} }
+    expect(response).to redirect_to('/users/sign_in')
```

Comment:
You could use the route helper if you want (`new_session_path` I think)

Reply:
new_user_session_path werks ;)